src/HonzaBotner.Discord.Services/WIPCommands/FunCommands.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DSharpPlus.SlashCommands;
+using HonzaBotner.Discord.Services.Extensions;
+
+namespace HonzaBotner.Discord.Services.Commands;
+
+public class FunCommands : ApplicationCommandModule
+{
+
+    [SlashCommand("choose", "Pick one of provided options")]
+    public async Task ChooseCommandAsync(
+        InteractionContext ctx,
+        [Option("options", "Options to choose from")]
+        string options,
+        [Option("delimiter", "Character that separates options. Default: \",\"")]
+        [MinimumLength(1), MaximumLength(1)]
+        string delimiter = ","
+        )
+    {
+        var answers = options.Split(delimiter, StringSplitOptions.TrimEntries & StringSplitOptions.RemoveEmptyEntries)
+            .Select(option => option.Trim().RemoveDiscordMentions())
+            .Where(option => option != "").ToArray();
+        if (answers.Length == 0)
+        {
+            await ctx.CreateResponseAsync("Nope");
+            return;
+        }
+        Random random = new();
+        var text = new StringBuilder("I picked: ");
+        int winNumber = random.Next(answers.Length);
+        string winner = answers[winNumber];
```

Comment:
We are mixing several method of variable declarations. Pick one and stick with it.